torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -534,6 +538,17 @@ class TensorExprFuser {
     }
   }
 
+  // No Ops in eager shouldn't be outputs of Fusion Groups because it
+  // will degrade perf and change aliasing relationships
+  static bool unexecutedEagerOp(Node* n) {
+    if (n->kind() != aten::to) {
+      return false;
+    }
+
+    return *n->input(0)->type()->expect<TensorType>() ==
```

Comment:
Can we optimize away these ops with const-prop or something?

Reply:
sadly not, because we only have the profiled type here. If it is an a fusion group it will be a No-op tho.